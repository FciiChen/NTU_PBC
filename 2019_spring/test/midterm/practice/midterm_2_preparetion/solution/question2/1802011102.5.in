WJ1, arc,or8TIa7t0Z,KoSwFYC2,N4R,4TM,ZhzmRjP,5rP9,GkFw0j
TMiKj8, TYSMezS,5315LnLro3,uoEpZJKIWh,dSq,VEYl,o1TCsLR6,2ZU7z,XdIB
ybW4KZC, PaKZy,mSmkvML,uOwl0p2,rdaT,NK8K,b1Cb,cxDad20nKW,4nhev
BREAK
WJ1 ,  Cs8
ybW4KZC ,  rjT37
TMiKj8 ,  tvos
BREAK
