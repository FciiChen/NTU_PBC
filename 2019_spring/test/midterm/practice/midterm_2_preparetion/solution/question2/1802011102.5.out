WJ1        arc        or8TIa7t0Z KoSwFYC2   N4R        4TM        ZhzmRjP    5rP9       GkFw0j     Cs8       
TMiKj8     TYSMezS    5315LnLro3 uoEpZJKIWh dSq        VEYl       o1TCsLR6   2ZU7z      XdIB       tvos      
ybW4KZC    PaKZy      mSmkvML    uOwl0p2    rdaT       NK8K       b1Cb       cxDad20nKW 4nhev      rjT37     
